spec/index.bs

```diff
@@ -1075,10 +1075,12 @@ allows additional usages of WebGPU that would have otherwise been invalid.
 
 <script type=idl>
 enum GPUExtensionName {
-    "texture-compression-bc",
+    "depth-clamping",
+    "depth24unorm-stencil8",
+    "depth32float-stencil8",
     "pipeline-statistics-query",
+    "texture-compression-bc",
     "timestamp-query",
```

Comment:
Though WebIDL [allows trailing commas](https://heycam.github.io/webidl/#index-prod-Enum), we've been avoiding them because WebKit is out-of-spec on this. Maybe we should just go ahead and diverge?

Reply:
IMHO that's fine, each browser vendor most likely will integrate WebIDL manually when bringing up the API and following its changes.